lib/LaTeXML/Package/article.cls.ltxml

```diff
@@ -101,5 +101,17 @@ DefMacro('\appendix', '\@appendix');
 # Actually we should be using section counter
 DefPrimitive('\@appendix', sub { startAppendices('section'); });
 
+#======================================================================
+DefPrimitiveI('\tiny',         undef, undef, font => { size => 5 });
+DefPrimitiveI('\scriptsize',   undef, undef, font => { size => 7 });
+DefPrimitiveI('\footnotesize', undef, undef, font => { size => 8 });
+DefPrimitiveI('\small',        undef, undef, font => { size => 9 });
+DefPrimitiveI('\normalsize',   undef, undef, font => { size => 10 });
+DefPrimitiveI('\large',        undef, undef, font => { size => 12 });
+DefPrimitiveI('\Large',        undef, undef, font => { size => 14.4 });
+DefPrimitiveI('\LARGE',        undef, undef, font => { size => 17.28 });
+DefPrimitiveI('\huge',         undef, undef, font => { size => 20.74 });
+DefPrimitiveI('\Huge',         undef, undef, font => { size => 29.8 });
+
```

Comment:
I should flag that I plan to open a related issue: `\tiny`, ..., `\Huge` are responsible for changing `\baselineskip`, and the fact that LaTeXML does not do that has very visible side effects with TikZ.